test/integration/shoots/maintenance/maintenance.go

```diff
@@ -71,7 +71,7 @@ import (
 )
 
 // TODO: this test is currently not executed, because it has to manipulate the CloudProfile. It should be refactor into
-// an envtest-style integration test, so that we can enable it again.
+//  an envtest-style integration test, so that we can enable it again.
```

Comment:
was this on purpose? Seems like we usually don't format this way https://github.com/gardener/gardener/blob/master/pkg/registry/core/project/strategy.go#L69

Reply:
same in the other file

Reply:
Yep. My IDE only shows it as a multiline `TODO` comment if it's indented.
I assume, this is a generally accepted formatting.